robottelo/api/utils.py

```diff
@@ -74,6 +76,92 @@ def upload_manifest(organization_id, manifest):
     )
 
 
+def publish_puppet_module(puppet_modules, repo_url, organization_id=None):
```

Comment:
This function misses coverage, but I think it would be difficult to write a test without mocking Nailgun client.

So or we write a proper testing (by implementing Nailgun mock)

or we explicitly say to coverage that we are noit going to cover those lines.

adding a `# pragma: no cover` comment to the scope

```python
def publish_puppet_module(puppet_modules, repo_url, organization_id=None):   # pragma: no cover

```